pkg/term/term_windows.go

```diff
@@ -1,12 +1,16 @@
 // +build windows
+
 package term
 
 import (
+	"fmt"
 	"io"
 	"os"
+	"os/signal"
 
 	"github.com/Sirupsen/logrus"
-	"github.com/docker/docker/pkg/term/winconsole"
+	"github.com/docker/docker/pkg/term/windows"
+	"github.com/docker/docker/vendor/src/github.com/Azure/go-ansiterm/winterm"
```

Comment:
Wouldn't this directly reference the Azure/go-ansiterm repo?  Shouldn't the references go through the vendor version in the Docker repo?  Is there some other mechanism I'm unaware of which intercepts these references?  Do you mean only this file should reference the Azure repo?

I'm ok with everything directly referencing the Azure repo, but I thought the Docker team preferred snapshots of vendor code tied to particular releases/tags/commits.